cc/runfiles/BUILD

```diff
@@ -1,7 +1,14 @@
+load("@rules_cc//cc:cc_library.bzl", "cc_library")
+
 licenses(["notice"])
 
-alias(
+cc_library(
     name = "runfiles",
-    actual = "@bazel_tools//tools/cpp/runfiles",
+    srcs = ["runfiles.cc"],
+    hdrs = ["runfiles.h"],
+    # Ensure that this header can be included from the same directory as the
```

Comment:
I think that would be a bit better. Single migration instead of 2 and it's hard to motivate the second migration.

Also setup is much simpler without `include_prefixes`, C++ rules don't need to create any symlinks.

Reply:
The natural prefix would be `cc/runfiles`, which I think may be too short. Making it `rules_cc/cc/runfiles` would require a symlink. What do you think?

Reply:
Should I also change the namespace to something like `rules_cc::runfiles`?